Source/DataSources/PolylineGeometryUpdater.js

```diff
@@ -535,6 +600,54 @@ define([
             return;
         }
 
+        // Synchronize with geometryUpdater for GroundPolylinePrimitive
+        geometryUpdater._clampToGround = Property.getValueOrDefault(polyline._clampToGround, time, false);
+        geometryUpdater._groundGeometryOptions.positions = positions;
+        geometryUpdater._groundGeometryOptions.width = Property.getValueOrDefault(polyline._width, time, 1);
+
+        var groundPrimitives = this._groundPrimitives;
+
+        if (defined(this._groundPolylinePrimitive)) {
+            groundPrimitives.remove(this._groundPolylinePrimitive); // destroys by default
+            this._groundPolylinePrimitive = undefined;
+        }
+
+        if (geometryUpdater.clampToGround) {
+            var that = this;
+
+            // Load terrain heights
+            if (!this._terrainHeightsReady) {
```

Comment:
Check `GroundPolylinePrimitive._initialized` or make a private function so this doesn't wait on the promise for every instance.